data/categories.txt

```diff
@@ -500,7 +501,7 @@ ro:bani
 es:dinero
 sv:pengar
 th:เงิน|atm
-tr:atm|3para
+tr:atm|3para|U+1F3E7|U+1F4B2|U+1F4B3|U+1F4B4|U+1F4B5|U+1F4B6|U+1F4B7
```

Comment:
All synonyms from en: are always used, so duplicating them here is not necessary.
```suggestion
tr:atm|3para
```